proto/teleport/lib/teleterm/v1/cluster.proto

```diff
@@ -141,6 +141,8 @@ message ACL {
   ResourceAccess recorded_sessions = 13;
   // active_sessions defines access to active sessions.
   ResourceAccess active_sessions = 14;
+  // review_requests defines the ability to review requests
+  optional bool review_requests = 15;
```

Comment:
I made this optional because the rest of the acl is optional (because of pointing to structs). If it wasn't optional here, then the type the frontend uses would require me to update [all sorts of tests spots ](https://github.com/gravitational/teleport/blob/avatus/review_request/web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputer/Setup.test.tsx#L118-L129) that are completely unrelated.

Reply:
I think it's better to fix the usages and keep the proto aligned with the implementation. From what I see, it's only 8 usages, so not that bad.
I'd add a new test helper, similar to `makeRootCluster` or `makeLoggedInUser`:
```ts
export const makeAcl = (
  props: Partial<ACL> = {},
) => ({
  recordedSessions: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  activeSessions: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  authConnectors: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  roles: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  users: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  trustedClusters: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  events: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  tokens: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  servers: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  apps: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  dbs: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  kubeservers: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  accessRequests: {
    list: true,
    read: true,
    edit: true,
    create: true,
    delete: true,
    use: true,
  },
  reviewRequests: true,
});
```

and then use it everywhere we make ACLs:
```ts
  const cluster = makeRootCluster({
    loggedInUser: makeLoggedInUser({
      acl: makeAcl({
        tokens: {
          create: true,
          list: true,
          read: true,
          edit: true,
          delete: true,
          use: true,
        },
      }),
    }),
  });
```

Reply:
Yeah thats much better. Updated